lib/exporters/jhipster_entity_exporter.js

```diff
@@ -124,7 +123,13 @@ function writeEntities(subFolder) {
   for (let i = 0, entityNames = Object.keys(configuration.entities); i < entityNames.length; i++) {
     const filePath = path.join(subFolder, toFilePath(entityNames[i]));
     const entity = updateEntityToGenerateWithExistingOne(filePath, configuration.entities[entityNames[i]]);
-    fs.writeFileSync(filePath, JSON.stringify(entity, null, 2).concat('\n'));
+    if (configuration.returnMergedEntity) {
+      // Set the updated entity back.
+      configuration.entities[entityNames[i]] = entity;
+    }
+    if (!configuration.skipEntityFilesGeneration) {
```

Comment:
Having a conditional that says "if we don't want to write entity files" in a function that's called `writeEntities` is weird, don't you think?
Having two functions makes more sense.

Reply:
I agree but if we follow the rules, this change should be a major release.

Reply:
No need, it's a private function. Had it been in the exports then yes.
How do you see it? Maybe we're not thinking about the same things :)

My idea is : `exportEntities` should call two functions. The first one being the "we want to write files to the disk" and the other "the flag is set, let's not write"

Reply:
I am thinking about the following approaches:
- Create a new mergeExistingEntityFromFile.
Duplicate current writeEntities removing the write part.
- Rename the function to mergeAndWriteEntities and don’t duplicate the loop